projects/demo/src/pages/recipes/card/examples/1-basic/component.ts

```diff
@@ -20,7 +20,7 @@ export class CardDocExample1 {
             ' ',
             ...Array(4).fill(/\d/),
             ' ',
-            ...Array(3).fill(/\d/),
+            ...Array(4).fill(/\d/),
```

Comment:
Hello!
Thank you for your contribution!

Could you provide us with the source where we can read that card number can be 20-characters length?

I've tried to ask Google about it. It says:
> Payment card numbers are composed of 8 to 19 digits

Source: [Wikipedia](https://en.wikipedia.org/wiki/Payment_card_number#:~:text=4%20References-,Structure,bank%20identification%20number%20(BIN))

Reply:
The existing documentation on `main` is correct. I somehow misinterpreted the placeholder text vs. the supported mask input and thought it should accept 4 digits at the end. 

I'll close out this PR, it isn't needed 🙂